.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,39 +1,17 @@
 <!--- Please review your changes in preview mode -->
 <!--- Provide a general summary of your changes in the Title above -->
 
-#### Bug/feature Description
-* *Clearly and concisely describe the problem/feature as it affects the customer (this cannot be empty)*
-* *Bugs: clearly describe the steps to easily reproduce the problem; you can include short illustrative excerpts from log files or terminal sessions*
-* *Bugs: if the issue is intermittent and not easily reproducible, describe the situation/scenarios/configurations where it was likely triggered*
-* *If there is an issue ID, link to it: [PP-XXXX](https://pbspro.atlassian.net/browse/PP-XXXX)*
+#### Bug or Feature Description
+* *Describe the problem, ideally from the customer's viewpoint*
```

Comment:
Now that these guidelines are just a single line in each section, I kind of feel like we should make these comments (which are invisible) in the template and not actual text that the developer has to erase. You can make them comments by surrounding the text by <!---  (refer to the Checklist section)